src/main/java/org/generationcp/middleware/domain/inventory/planting/PlantingRequestDto.java

```diff
@@ -18,6 +18,10 @@ public static class WithdrawalInstruction {
 
 		private boolean withdrawAllAvailableBalance;
 
+		private boolean withdrawUsingEntryDetail;
+
+		private Integer entryDetailsVariableId;
```

Comment:
rename to entryDetailVariableId @cuenyad

Reply:
done.